src/lib/autoinstall/autosetup_helpers.rb

```diff
@@ -197,8 +197,44 @@ def profile_checker
       Y2Autoinstallation::XmlChecks.instance
     end
 
+    # Invokes autoyast setup for firewall
+    def autosetup_firewall
+      return if !Yast::Profile.current["firewall"]
```

Comment:
We should not directly access the profile values. Some time ago we had a card which focused on removing that: https://trello.com/c/kCfh7Ylo/1941-5-drop-references-to-profilecurrent

we should avoid adding that back. Please see the linked pull requests for reference how to replace that.

Reply:
well ... using ```Profile.current``` is quite often in AY package:

```
# fgrep -r Profile.current src/ | wc -l
131
```

However, I'll give it a try whether i can live without that here.

Reply:
Ah, this is the AutoYaST package itself, using `Profile.current` there is probably OK. We do not want to use it from the other modules....